config.json

```diff
@@ -41,9 +41,9 @@
     "Firefox ESR"
   ],
   "settings": {
-    "homepage_new_products_count": 12,
-    "homepage_featured_products_count": 8,
-    "homepage_top_products_count": 8,
+    "homepage_new_products_count": 5,
```

Comment:
Let’s keep this consistent with the other and make it 4 :tropical_drink:

Reply:
We should confirm this change does not break any tests which are looking for X products

Reply:
I made it five because we use a carousel for this element and I wanted to show off its capabilities.

Reply:
$this->assertCount(12, $productsFromUI, 'Expected to see 12 new products');
NewProductsUITest.php

Reply:
Yep, we'll update the tests once this is published.